charmverse/app.charmverse.io
# Gnosis safe csv airdrop

Integrated the option on the bounty list page to export to CSV in Gnosis Safe Airdrop format information pertaining to completed bounties.

![export to CSV](https://user-images.githubusercontent.com/12700927/158894763-3895a787-2b18-4724-bb76-014a90da820c.png)

[Gnosis Safe Airdrop.csv](https://github.com/charmverse/app.charmverse.io/files/8292514/Gnosis.Safe.Airdrop.csv)

Test steps:

1. Go to **/bounties**.
2. Make sure there are some completed bounties.
3. Notice the **Export to CSV** button.
4. Click the button.
5. Check that the generated CSV is valid. If the denomination is a native token, the **token_address** should be blank.
Additional information: https://github.com/bh2smith/safe-airdrop#preparing-a-transfer-file
